packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -118,6 +120,9 @@ library TaikoData {
         uint64 proposedAt; // timestamp
         uint64 proposedIn; // L1 block number, required/used by node/client.
         uint32 nextTransitionId;
+        // The ID of the transaction that is used to verify this block. However, if
```

Comment:
@RogerLamTd Previous for every block verified, this verifiedTransitionId will be non-zero, now, only the last block verified in a batch will have verifiedTransitionId non-zero. Do you think you can update related documentation?

Reply:
> @RogerLamTd Previous for every block verified, this verifiedTransitionId will be non-zero, now, only the last block verified in a batch will have verifiedTransitionId non-zero. Do you think you can update related documentation?

will update description of how we verify blocks in batches now in documentation and tag this PR